bench/workgen/workgen.cpp

```diff
@@ -456,63 +486,74 @@ WorkloadRunner::start_tables_drop(WT_CONNECTION *conn)
             continue;
         }
 
-        for (int drops = 0; drops < _workload->options.drop_count; ++drops) {
-            std::vector<std::string> drop_uris;
-            // The data structures for the dynamic table set are protected by a mutex.
-            {
-                const std::lock_guard<std::mutex> lock(*icontext->_dyn_mutex);
-
-                // Walk the list and select the first that is not already in the delete list.
-                for (auto const &it : icontext->_dyn_tint) {
-                    if (icontext->_dyn_table_runtime[it.second]._pending_delete) {
-                        continue;
-                    } else {
-                        VERBOSE(*_workload, "Marking pending removal for: " << it.first);
-                        icontext->_dyn_table_runtime[it.second]._pending_delete = true;
-                        tables_delete.push_back(it.first);
-                        break;
-                    }
-                }
+        std::vector<std::string> drop_files; // Files ready to be dropped.
+        // The data structures for the dynamic table set are protected by a mutex.
+        {
+            const std::lock_guard<std::mutex> lock(*icontext->_dyn_mutex);
 
+            /*
+             * When dropping consider how many dynamic tables we have left, and how many are already
```

Comment:
Comma should be after "dropping" rather than "left":
```
/*
 * When dropping, consider how many dynamic
 * tables we have left and how many are already
 * marked for deletion.
*/
```

Reply:
Fixed.